client/state/memberships/product-list/selectors.js

```diff
@@ -1,13 +1,8 @@
-/**
- * External dependencies
- */
-import { get } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import 'calypso/state/memberships/init';
 
 export function getProductsForSiteId( state, siteId ) {
-	return get( state, [ 'memberships', 'productList', 'items', siteId ] );
+	return state.memberships?.productList.items[ siteId ] ?? [];
```

Comment:
The selector returns a different instance of the empty array on every call, which may cause unneeded rerenders. A good practice is to extract the empty list value:
```js
const EMPTY_LIST = [];
const getProducts = state => state.items[ siteId ] ?? EMPTY_LIST;
```

Reply:
Good call! I was too hasty to fix it 😅  Fixed in 19f0427ff8b14647c44a6208f3072dd6e26b868d